addons/sale_expense/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,57 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class HrExpenseSheet(models.Model):
+    _inherit = "hr.expense.sheet"
+
+    def _get_sale_order_lines(self):
```

Comment:
I think the readability of this function can be improved...
try variable names like `aml_to_so_map`, `potential_sol_map`
or maybe the docstring could be more descriptive: ".... it does this by..."
or maybe just comment the code

Reply:
Improved (I hope so)